<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructureRythm</class>
 <widget class="QWidget" name="StructureRythm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 211, 239);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QPushButton" name="pushButtonRonde">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/NoteRondeModifié.png</normaloff>:/images/NoteRondeModifié.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNoir">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/NoteNoirModifier.png</normaloff>:/images/NoteNoirModifier.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNoirP">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/NoteNoirPModifier.png</normaloff>:/images/NoteNoirPModifier.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBlanche">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/NoteBlancheModifier.png</normaloff>:/images/NoteBlancheModifier.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBLancheP">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/NoteBlanchePModifier.png</normaloff>:/images/NoteBlanchePModifier.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonValider">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>valider</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/valide.png</normaloff>:/images/valide.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>593</x>
     <y>273</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/reset.png</normaloff>:/images/reset.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCroche">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/NoteCrocheModifier.png</normaloff>:/images/NoteCrocheModifier.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonValider</sender>
   <signal>clicked()</signal>
   <receiver>StructureRythm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
